stable/wordpress/templates/ingress.yaml

```diff
@@ -17,6 +17,15 @@ metadata:
     {{- end }}
 spec:
   rules:
+  {{- if .Values.ingress.hostname }}
+  - host: {{ .Values.ingress.hostname }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: "{{ template "wordpress.fullname" $ }}"
+          servicePort: http
+  {{- end }}
```

Comment:
I would add some kind of validations so we print a warning in the NOTES.txt when both `ingress.hostname` and `ingress.hosts` parameters are set.

Check how we do it in other charts. For instance:

- https://github.com/bitnami/charts/blob/master/bitnami/minio/templates/NOTES.txt#L70
- https://github.com/bitnami/charts/blob/master/bitnami/minio/templates/_helpers.tpl#L121

Reply:
Well, in this case it's okay to set up both `ingress.hostname` and `ingress.hosts`. I changed the comment to:
_The list of **additional** hostnames to be covered with this ingress record_